packages/core/src/utilities/background-color/background-color.example.html

```diff
@@ -1,19 +1,25 @@
-<% var groups = [ ['primary', 'primary-darker', 'primary-darkest'], ['primary-alt',
-'primary-alt-dark', 'primary-alt-darkest', 'primary-alt-light', 'primary-alt-lightest'], ['gray',
-'gray-dark', 'gray-light', 'gray-lighter', 'gray-lightest'], ['gold', 'gold-light', 'gold-lighter',
-'gold-lightest'], ['warn', 'warn-light', 'warn-lighter', 'warn-lightest'], ['green', 'green-light',
-'green-lighter', 'green-lightest'], ['success', 'success-light', 'success-lighter',
-'success-lightest'], ['secondary', 'secondary-dark', 'secondary-darkest', 'secondary-light',
-'secondary-lightest'], ['error', 'error-dark', 'error-darkest', 'error-light', 'error-lighter',
-'error-lightest'], ['base', 'white', 'background', 'background-inverse', 'transparent'] ]; -%> <%
-groups.forEach(group => { -%>
-<section class="ds-l-row ds-u-margin-x--0 ds-u-margin-bottom--3">
-  <% group.forEach(color => { -%>
-  <article class="ds-l-col--12 ds-l-sm-col--6 ds-u-padding--0 ds-u-margin-bottom--2">
-    <code class="ds-u-margin-bottom--1">.ds-u-fill--<%= color %></code>
-    <span class="ds-u-color--muted js-swatch-hex">#</span>
-    <div class="c-swatch__preview c-swatch__preview--condensed ds-u-fill--<%= color %>"></div>
-  </article>
-  <% }) -%>
-</section>
-<% }) %>
+<div class="example--list">
+  <% var groups = { 'Primary colors': ['primary', 'primary-darker', 'primary-darkest', 'black',
+  'base', 'gray-dark', 'gray-light', 'white'], 'Secondary colors': ['primary-alt',
+  'primary-alt-dark', 'primary-alt-darkest', 'primary-alt-light', 'primary-alt-lightest'],
+  'Background colors': ['background', 'background-inverse','gray-dark', 'gray', 'gray-lighter',
```

Comment:
What do you think of moving `transparent` to "Additional colors"? Thats where it is located in the Color palette page and I dont think it makes as much sense to have a transparent bg